lib/customer.rb

```diff
@@ -0,0 +1,60 @@
+require 'csv'
+
+class Customer
+
+  def initialize(id, email, address)
+  @id = id
+  @email = email
+  @address = address
+  end
+
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
These should be attr_reader instead of attr_accessor because you don't need to give users of your class the ability to directly change the instance variables.  A reader is more than enough access.